test/src/module/protocol/protocolTest.c

```diff
@@ -434,7 +434,7 @@ testRun(void)
 
         MEM_CONTEXT_TEMP_BEGIN()
         {
-            TEST_ASSIGN(command, protocolCommandNew(strNew("command1")), "create command");
+            TEST_ASSIGN(command, protocolCommandNew(strIdFromZ(stringIdBit6, "cmd1")), "create command");
```

Comment:
Given src/protocol/server.c line 134, I thought only stringIdBit5 was valid for protocol commands - how is it you can use stringIdBit6?

Reply:
Originally I had not added any numbers to 5-bit so I had made these 6-bit. It works because they are value 5-bit strings and they get converted to a string in the protocol layer. Clearer just to make them 5-bit so I did that in d8902130e0.